modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -245,6 +247,119 @@ The other two commands are:
 
 TIP: See xref:configuration/server/background_jobs_configuration.adoc[background jobs configuration] to learn more.
 
+== Managing Background Jobs
+
+Use the `background:queue` command to manage background jobs.
+
+[source,console]
+----
+background:queue
+ background:queue:delete     Delete a job from the queue
+ background:queue:execute    Run a single background job from the queue
+ background:queue:status     List queue status
+----
+
+=== Deleting a Backgroundjob
+
+The command `background:queue:delete` deletes a queued background job.
+It requires the job id of the job to be deleted.
+
+....
+background:queue:delete <Job ID>
+....
+
+==== Arguments
+
+[width="100%",cols="20%,70%",]
```

Comment:
There's no need, afaik, for the `width` parameter, unless you want the table to be less than 100%.

Reply:
This is true, but we have that on all the other tables.
For the sake of the same look and feel I kept it that way.